src/Features/Core/Portable/DocumentationComments/DocumentationCommentProposal.cs

```diff
@@ -11,13 +11,16 @@ namespace Microsoft.CodeAnalysis.DocumentationComments;
 /// </summary>
 internal sealed record DocumentationCommentProposal
 {
-    public string SymbolToAnalyze { get; }
+    public Document Document { get; }
+
+    public SyntaxNode MemberNode { get; }
```

Comment:
document what sort of member node this is?

type decls?  member decls?  anything else?  is it always something that allows doc comments:?